.github/workflows/doc.yml

```diff
@@ -17,10 +17,10 @@ jobs:
         uses: xu-cheng/latex-action@v2
         with:
           root_file: masterdoc.tex
-          working_directory: doc
+          working_directory: doc/latex
```

Comment:
I think you may also want to change https://github.com/UCL/TDMS/blob/50f6fe45eb880a578a3acb9979f4a5203b49c08c/.github/workflows/doc.yml#L7 to `'doc/latex/**'`

Reply:
Weirdly, it worked like that in testing. But it also works with the missing `/latex/`.